kajas/src/app/profile/profile.component.ts

```diff
@@ -10,109 +11,123 @@ import axios from 'axios';
   styleUrls: ['./profile.component.css']
 })
 export class ProfileComponent implements OnInit {
-  firstName = this.sessionStorage.get('first_name') || '';
-  lastName = this.sessionStorage.get('last_name') || '';
-  country = this.sessionStorage.get('country') || '';
-  city = this.sessionStorage.get('city') || '';
-  bio = this.sessionStorage.get('bio') || '';
+  firstName: string = '';
+  lastName: string = '';
+  country: string = '';
+  city: string = '';
+  bio: string = '';
   profile: string = '';
-  linkedin = this.sessionStorage.get('linkedin') || '';
-  facebook = this.sessionStorage.get('facebook') || '';
-  instagram = this.sessionStorage.get('instagram') || '';
-  website = this.sessionStorage.get('website') || '';
-  kajasLink = this.sessionStorage.get('kajas_link') || '';
+  linkedin: string = '';
+  facebook: string = '';
+  instagram: string = '';
+  website: string = '';
+  kajasLink: string = '';
   artworks: any[] = [];
-  message = '';
-  showDeleteButton = false;
-  showConfirmModal = false;
-  showMessageModal = false; 
-  modalMessage = '';
+  message: string = '';
```

Comment:
### Take advantage of TS inference type. If declaring a primitive variable that has an initial value, you don't  have to specify its type.
[https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g26feb3ebb2b_0_0](url)


`  message = '';
  showDeleteButton = false;
  showConfirmModal = false;
  showMessageModal = false; 
  modalMessage = '';
`